Unable to install jspm dependency

Npm install crash when install jspm

```console
npm ERR! git rev-list -n1 6c85e03c9bfee63bf17408b483b15b444b7f1ada: fatal: bad object 6c85e03c9bfee63bf17408b483b15b444b7f1ada
```

Node Version: 4.4.4
Npm Version: 3.9.2

Yeoman Generator Choices: 
```console
? Which JavaScript framework do you want? React
? Which module management do you want? SystemJS with JSPM
? Which JS preprocessor do you want? ES2015 today with Babel
? Which CSS preprocessor do you want? SASS
? Do you want a sample app? A working landing page
```

I reproduce this problem with Angular2 as JavaScript framework

[EDIT] 

Use **"jspm": "^0.17.0-beta.15"** instead of **"jspm": "jspm/jspm#6c85e03c9bfee63bf17408b483b15b444b7f1ada"** correct the problem.

JSPM team released 0.17.0-beta.15 today https://github.com/jspm/jspm-cli/releases/tag/0.17.0-beta.15

--- a/generators/app/index.js
+++ b/generators/app/index.js
@@ -43,8 +43,8 @@ module.exports = fountain.Base.extend({
 
       this.mergeJson('package.json', {
         devDependencies: {
-          'jspm': 'jspm/jspm#6c85e03c9bfee63bf17408b483b15b444b7f1ada',
-          'systemjs-builder': '^0.15.15',
+          'jspm': '^0.17.0-beta.15',
+          'systemjs-builder': '^0.15.18',
           'gulp-replace': '^0.5.4'
         },
         scripts: {

--- a/generators/app/index.js
+++ b/generators/app/index.js
@@ -43,8 +43,8 @@ module.exports = fountain.Base.extend({
 
       this.mergeJson('package.json', {
         devDependencies: {
-          'jspm': '^0.17.0-beta.15',
-          'systemjs-builder': '^0.15.18',
+          'jspm': '0.17.0-beta.15',
+          'systemjs-builder': '0.15.18',
           'gulp-replace': '^0.5.4'
         },
         scripts: {

